testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -55,7 +55,17 @@ def test_with_statement(self):
             err_msg="with_statement: DCDReader does not read all frames")
 
 
-class TestDCDReader(_TestDCD):
+class TestDCDReader(object):
```

Comment:
This change makes the class `_TestDCD` superfluous so you can delete it (deleting code is good!).

Then also remove it from `TestDCDCorrel(_TestDCD)` and just derive from `TestCase`.